### PR TITLE
Added Croud CLI docs and restructured a little bit

### PR DESCRIPTION
**Pull request by alijared**
I split the docs up so that there are two separate sub-pages for console and croud

### CODE REVIEW

File: docs/cli.txt [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+|                 |                     | with resources or logging in/out. **Note:** this  |
+|                 | * prod              | is only used by developers                        |
+|                 | * dev               |                                                   |
+|                 |                     |                                                   |
++-----------------+---------------------+---------------------------------------------------+
+
+Me
+==
+
+The `me` command prints the username and email address of the currently authenticated user\
```
Comment (m-kharbat) [inline docs/cli.txt]: extra` \` ?

Comment (alijared) [reply docs/cli.txt]: You're right, thanks!

**Comment by mechanomi** [comment]
@alijared I have added a commit to your branch with some edits. let's pass this back and forth a few times between me and you. once we're done, we can squash the whole thing down into one commit

please take a look at the changes I've made

some requests:

- please add information about how you can install the CLI tool

- please take a look at https://crate.io/docs/crate/reference/en/latest/run.html and make changes that bring your CLI docs inline with how this doc is structured. i.e., maybe add an intro section. also please update the "Usage" section I added. maybe rename it to "Command-Line Options" like the doc I just linked to? I'm not sure. but basically, this should be analogous to what you'd get if you ran ``--help`` on a command. if that makes sense?

- I have converted some of your examples into example blocks with syntax highlighting. but I have ot fully integrated these, or made them consistent. please take a look at this and make edits to the surrounding text

- the organizations command only lists one subcommand. is this correct?

- please convert the subcommand tables into subsections (so that they show up in the table of contents)

as mentioned: this first edit is not meant to fix everything. I've made some basic changes to the structure and presentation as an indication and I am passing back to you to take a look and add a third commit with your edits

once you've done this, I'll take a look and do some more edits. and in this way, we can pass the doc back and forth and collaborate on it that way

**Comment by alijared** [comment]
@mechanomi I've made the changes you requested:

- installation section updated
- usage section updated
- the organizations command does only have one sub-command now, yes
- converted sub-command tables into subsections

**Comment by mechanomi** [comment]
superceded by https://github.com/crate/croud/pull/56

**Pull request closed without merging**